src/push/index.ts

```diff
@@ -49,24 +49,42 @@ import type { PushOptions, ProjectSettings } from '../common_types';
 import { findSyntheticsConfig, readConfig } from '../config';
 
 export async function push(monitors: Monitor[], options: PushOptions) {
-  if (monitors.length === 0) {
-    throw 'No Monitors found';
-  }
-  const duplicates = trackDuplicates(monitors);
-  if (duplicates.size > 0) {
-    throw error(formatDuplicateError(duplicates));
-  }
+  let schemas: MonitorSchema[] = [];
+  if (monitors.length > 0) {
+    const duplicates = trackDuplicates(monitors);
+    if (duplicates.size > 0) {
+      throw error(formatDuplicateError(duplicates));
+    }
 
-  progress(`preparing all monitors`);
-  const schemas = await buildMonitorSchema(monitors);
+    progress(`preparing all monitors`);
+    schemas = await buildMonitorSchema(monitors);
 
-  progress(`creating all monitors`);
-  for (const schema of schemas) {
-    await pushMonitors({ schemas: [schema], keepStale: true, options });
-  }
+    progress(`creating all monitors`);
+    for (const schema of schemas) {
+      await pushMonitors({ schemas: [schema], keepStale: true, options });
+    }
+  } else {
+    write('');
+    // Makes testing easier with overrides
+    let deleteAll = false;
+    if (process.env.TEST_OVERRIDE != null) {
+      deleteAll = Boolean(process.env.TEST_OVERRIDE);
+    } else {
+      ({ deleteAll } = await prompt<{ deleteAll: boolean }>({
+        type: 'confirm',
+        name: 'deleteAll',
+        message: `Pushing without any monitors would delete all monitors associated with the project.\n Do you want to continue?`,
```

Comment:
```suggestion
        message: `Pushing without any monitors will delete all monitors associated with the project.\n Do you want to continue?`,
```

We should be more definitive with phrasing here